cli/azd/cmd/testdata/TestUsage-azd-config-get-key.snap

```diff
@@ -0,0 +1,18 @@
+
+Look for the key within the configuration and returns its value.
+
+Usage
+  azd config get-key <key> [flags]
+
+Flags
+        --docs 	: Opens the documentation for azd config get-key in your web browser.
+    -h, --help 	: Gets help for get-key.
+
+Global Flags
+    -C, --cwd string 	: Sets the current working directory.
+        --debug      	: Enables debugging and diagnostics logging.
+        --no-prompt  	: Accepts the default value instead of prompting, or it fails if there is no default.
+
+Find a bug? Want to let us know how we're doing? Fill out this brief survey: https://aka.ms/azure-dev/hats.
```

Comment:
nit: I don't think we need this since we aren't taking the `azd config get-key` command right now.